### ISSUE TITLE
Switch color doesn't match with App's Theme

### ISSUE THREAD

**Issue by SanjaySargam** [RocketChat/Rocket.Chat.ReactNative#5426]
### Environment Information:

- Rocket.Chat Server Version: 6.5.0
- Rocket.Chat App Version: 4.44.0+
- Device Name: Realme 9
- OS Version: Android

### Steps to reproduce:

1. Go to Display from DrawerNavigation
2. Turn on Avatars Switch

### Expected behavior:

The color of the switch should match with App's theme

### Actual behavior:

The switch color doesn't match with App's Theme

### Additional context:
![WhatsApp Image 2023-12-23 at 5 46 24 PM](https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/65113071/6dd0e822-36a1-41b9-9fa1-8dc44b10c925)

**Comment by SanjaySargam** [comment]
If this issue is accepted then I'd love to work on this

**Comment by GuptaPrisha** [comment]
Hey @SanjaySargam, I would like to know the current status of the resolution of this issue from you. Are you still working on it?

**Comment by SanjaySargam** [comment]
Hi Hey @GuptaPrisha . I already made a PR. 
You can work on other issues.

**Comment by GuptaPrisha** [comment]
Actually I have worked on this issue and completed it with the `jest` tests too.
Are you willing to collaborate on this issue? I can see your tests failing in the PR you made.

**Comment by SanjaySargam** [comment]
`There are several Switch on the app. Also I can see the track color is still the same. Can you create a reusable Switch and apply everywhere?`

@GuptaPrisha Did you consider this

**Comment by GuptaPrisha** [comment]
@SanjaySargam Yes! I made it reusable and adaptable. I made sure that it works well with other components which are already using the react-native's switch component.

Also, I applied it everywhere the native switch was getting used.

**Comment by SanjaySargam** [comment]
But I already made a PR. I will check why the jest is failing and resolve it.
And from the next time, check whether anyone is assigned to an issue or made a PR and then start working on it.

**Comment by jsathu07** [comment]
@SanjaySargam separate 'react-native' these kinds of imports from path imports ('../../') and add a line between them then lint will pass
